ceph_deploy/hosts/fedora/__init__.py

```diff
@@ -18,7 +18,7 @@ def choose_init():
 
     Returns the name of a init system (upstart, sysvinit ...).
     """
-    return 'sysvinit'
+    return 'systemd'
```

Comment:
Do we really want to switch this over wholesale?

F20 was EOL'd on 6/23, but we do still have packages for F20 in rpm-hammer.  We never built F21 packages.

My concern is that if we took this is in as-is, and I push a newer version of ceph-deploy into rpm-hammer, then ceph-deploy won't be able to work with the F20 packages that are also present in rpm-hammer.

I'm not sure if this is an issue or not.